src/server/setup.ts

```diff
@@ -86,8 +86,13 @@ export namespace Setup {
 
         passport.deserializeUser((userId: string, done) =>
             People.getById(userId)
-                .then(p => done(null, p))
-                .catch(e => done(e, null)));
+                .then(p => {
+                    if (p)
+                        done(null, p);
+                    else
+                        throw new Error('User ID not found');
```

Comment:
Dit werkt niet uit mijn ervaring. Als je in deze functie een error gooit dan krijg de client niet de goede response en wordt het cookie niet verwijderd. Je moet done(null, false) returnen (zie https://github.com/jaredhanson/passport/issues/6#issuecomment-4857287)